llvm-runtimes/libgcc/libgcc-20.0.0.9999.ebuild

```
@@ -0,0 +1,153 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit cmake crossdev flag-o-matic llvm.org llvm-utils python-any-r1
+inherit toolchain-funcs
+
+DESCRIPTION="Compiler runtime library for clang, compatible with libgcc_s"
+HOMEPAGE="https://llvm.org/"
+
+LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
+SLOT="0"
+IUSE="+abi_x86_32 abi_x86_64 debug test"
+
+DEPEND="
+	~llvm-runtimes/libunwind-${PV}[static-libs]
+	!!sys-devel/gcc
+"
```

```suggestion
DEPEND="
	~llvm-runtimes/libunwind-${PV}[static-libs]
"
RDEPEND="
	${DEPEND}
	!sys-devel/gcc
"
```

I don't think you need a hard blocker on GCC? I mean, this package should build just fine if GCC is around, right?

---

Yes, true, enforcing runtime dep conflict be enough. This is also going to be better for crossdev.

---

Done